lib/Service/AvatarService.php

```diff
@@ -200,7 +200,7 @@ public function getAvatarUrl(Room $room): string {
 		];
 
 		$avatarVersion = $room->getAvatar();
-		if ($avatarVersion) {
+		if ($avatarVersion !== '') {
```

Comment:
To do this we will need set a default value in database to empty, for now, the default value is null.

Reply:
I fixed that. And also the Manager was already casting to string anyway.